llvm/test/Transforms/InstCombine/pr85536.ll

```diff
@@ -0,0 +1,44 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
+; RUN: opt -S -passes=instcombine < %s | FileCheck %s
+
+define i8 @test_drop_noundef(i1 %cond, i8 %val) {
```

Comment:
Can you please add the test next to other tests for this fold? I think they're in intrinsic-select.ll.